library/azure_rm_cosmosdbaccount.py

```diff
@@ -0,0 +1,488 @@
+            - Required when C(state) is I(present).
+        type: list
+        suboptions:
+            location_name:
+                description:
+                    - The name of the region.
+            failover_priority:
+                description:
+                    - "The failover priority of the region. A failover priority of 0 indicates a write region. The maximum value for a failover priority =
+                       (total number of regions - 1). Failover priority values must be unique for each of the regions in which the database account exists."
+    database_account_offer_type:
+        description:
+            - Required when C(state) is I(present).
+    ip_range_filter:
+        description:
+            - "Cosmos DB Firewall Support: This value specifies the set of IP addresses or IP address ranges in CIDR form to be included as the allowed list
+               of client IPs for a given database account. IP addresses/ranges must be comma separated and must not contain any spaces."
+    is_virtual_network_filter_enabled:
+        description:
+            - Flag to indicate whether to enable/disable Virtual Network ACL rules.
+    enable_automatic_failover:
+        description:
+            - "Enables automatic failover of the write region in the rare event that the region is unavailable due to an outage. Automatic failover will
+               result in a new write region for the account and is chosen based on the failover priorities configured for the account."
+        type: bool
+    capabilities:
+        description:
+            - List of Cosmos DB capabilities for the account
+        type: list
+        suboptions:
+            name:
+                description:
+                    - "Name of the Cosmos DB capability. For example, 'name': 'EnableCassandra'. Current values also include 'EnableTable' and
+                       'EnableGremlin'."
+    virtual_network_rules:
+        description:
+            - List of Virtual Network ACL rules configured for the Cosmos DB account.
+        type: list
+        suboptions:
+            id:
+                description:
+                    - "Resource ID of a subnet, for example:
+                       /subscriptions/{subscriptionId}/resourceGroups/{groupName}/providers/Microsoft.Network/virtualNetworks/{virtualNetworkName}/subnets/{
+                      subnetName}."
+    enable_multiple_write_locations:
+        description:
+            - Enables the account to write in multiple I(locations)
+    state:
+      description:
+        - Assert the state of the Database Account.
+        - Use 'present' to create or update an Database Account and 'absent' to delete it.
+      default: present
+      choices:
+        - absent
+        - present
+
+extends_documentation_fragment:
+    - azure
+    - azure_tags
+
+author:
+    - "Zim Kalinowski (@zikalino)"
+
+'''
+
+EXAMPLES = '''
+  - name: Create (or update) Database Account
```

Comment:
sample to create documentdb, graph account,  table etc

Reply:
added one more sample with more params...